Benchmarks/Benchmarks.csproj

```diff
@@ -0,0 +1,199 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the purpose of having this project after migrating?
Seems it should be deleted now.

Reply:
There are still lots of bottlenecks that can be implemented differently and have different performance. The only proper way to compare implementations is to benchmark them.

The project is a tool to run such benchmarks. Everything required to run a bench is already installed and set up, so it is now easy for new contributors to write benchmarks.

Reply:
No, let's put it somewhere else (you can put it to your personal repo).
You can easily find slow code using profilers/test/someting else.

Reply:
> You can easily find slow code using profilers/test/someting else.

It's not true. There are a lot of traps and pitfalls in benchmark testing which are appropriately handled by BenchmarkDotnet.

>  let's put it somewhere else 
"Put-it-somewhere-else"-approach leads forking the repo and living on a forked version.

Please provide a proper reason why this project should not be in solution (other than "it should be deleted").

Reply:
Ok, but let's move it from separate project to Tests.csporj/Benchmarks, like in https://github.com/vostok/commons.collections/blob/master/Vostok.Commons.Collections.Tests/ImmutableArrayDictionary_Benchmark.cs.

Now you can't run one of them and have strange console app.

Reply:
done